deploy/olm-catalog/ibm-block-csi-operator/1.0.0/ibm-block-csi-operator.v1.0.0.clusterserviceversion.yaml-review

```diff
@@ -0,0 +1,493 @@
+      - RHCOS 4.1 (x86 architecture)
+
+    ## Pre-requisites
+    
+    ### Worker nodes preparation
+    Perform these steps for each worker node in OpenShift:
+    
+    ### 1. Install Linux packages to ensure Fibre Channel and iSCSI connectivity
+    Skip this step, if the packages are already installed.
+    
+    RHEL 7.x:
+    ```bash
+    $ yum -y install sg3_utils
+    $ yum -y install iscsi-initiator-utils   # only if iSCSI connectivity is required
+    $ yum -y install xfsprogs                # Only if xfs filesystem is required.
+    ```
+    
+    #### 2. Configure Linux multipath devices on the host.
+    Create and set the relevant storage system parameters in the `/etc/multipath.conf` file.
+    You can also use the default `multipath.conf` file located in the `/usr/share/doc/device-mapper-multipath-*` directory.
+    Verify that the `systemctl status multipathd` output indicates that the multipath status is active and error-free.
+    
+    RHEL 7.x:
+    ```bash
+    $ yum install device-mapper-multipath
+    $ modprobe dm-multipath
+    $ systemctl start multipathd
+    $ systemctl status multipathd
+    $ multipath -ll
+    ```
+    
+    Important: When configuring Linux multipath devices, verify that the `find_multipaths` parameter in the `multipath.conf` file is disabled.
+      - RHEL 7.x: Remove the `find_multipaths yes` string from the `multipath.conf` file.
+    
+    #### 3. Configure storage system connectivity.
+    3.1. Define the hostname of each OpenShift worker node on the relevant storage systems with the valid WWPN or IQN of the node.
+    
+    3.2. For Fiber Chanel, configure the relevant zoning from the storage to the host.
+    
+    3.3. For iSCSI, perform these three steps.
+    
+    3.3.1. Make sure that the login used to log in to the iSCSI targets is permanent and remains available after a reboot of the worker node. To do this, verify that the node.startup in the /etc/iscsi/iscsid.conf file is set to automatic. If not, set it as required and then restart the iscsid service `$> service iscsid restart`.
+    
+    3.3.2. Discover and log into at least two iSCSI targets on the relevant storage
+    systems.
+    
+    ```bash
+    $ iscsiadm -m discoverydb -t st -p ${storage system iSCSI port IP}:3260
+    --discover
+    $ iscsiadm -m node -p ${storage system iSCSI port IP/hostname} --login
+    ```
+    
+    3.3.3. Verify that the login was successful and display all targets that you logged in. The portal value must be the iSCSI target IP address.
+    
+    ```bash
+    $ iscsiadm -m session --rescan
+    Rescanning session [sid: 1, target: {storage system IQN},
+    portal: {storage system iSCSI port IP},{port number}
+    ```
+
+    ### Install CSIDriver CRD
+    Enabling CSIDriver on OpenShift
+    1. Enable Technology Preview features using feature gates, more details, please refer to https://docs.openshift.com/container-platform/4.1/nodes/clusters/nodes-cluster-enabling-features.html
```

Comment:
1. Enable Technology Preview features using feature gates. For more details, refer to https://docs.openshift.com/container-platform/4.1/nodes/clusters/nodes-cluster-enabling-features.html.